packages/react-interactions/events/src/dom/PressLegacy.js

```diff
@@ -803,46 +801,7 @@ const pressResponderImpl = {
             }
           }
 
-          const pressTarget = state.pressTarget;
           dispatchPressEndEvents(event, context, props, state);
-          const onPress = props.onPress;
-
-          if (pressTarget !== null && isFunction(onPress)) {
-            if (
-              !isKeyboardEvent &&
-              pressTarget !== null &&
-              target !== null &&
-              !targetIsDocument(pressTarget)
-            ) {
-              if (
-                pointerType === 'mouse' &&
-                context.isTargetWithinNode(target, pressTarget)
-              ) {
-                state.isPressWithinResponderRegion = true;
-              } else {
-                // If the event target isn't within the press target, check if we're still
-                // within the responder region. The region may have changed if the
-                // element's layout was modified after activation.
-                updateIsPressWithinResponderRegion(
```

Comment:
I think this function's definition can be deleted from the module

Reply:
state.isPressWithinResponderRegion is used for tracking isPressed and to fire pressStart, pressEnd and pressMove events, so I think it needs to stay for now.

Reply:
Yeah it's doesn't need to be removed to get `onPress` backed by `click`. This is only used to reactivate buttons when a down-pointer moves in and out of the hit rect. We already stopped doing that in [Tap](https://github.com/facebook/react/blob/35867e8094367fdebc4711940eb709454047c2d4/packages/react-interactions/events/src/dom/Tap.js (the intended replacement for Press) because it's work that we can avoid doing. But no need to get involved in that, we can remove this separately.